python/ray/rllib/ppo/ppo.py

```diff
@@ -101,15 +103,32 @@ def default_resource_request(cls, config):
             extra_gpu=cf["num_gpus_per_worker"] * cf["num_workers"])
 
     def _init(self):
-        self.global_step = 0
-        self.local_evaluator = PPOEvaluator(
-            self.env_creator, self.config, self.logdir, False)
-        RemotePPOEvaluator = ray.remote(
+        def session_creator():
+            return tf.Session(
+                config=tf.ConfigProto(**self.config["tf_session_args"]))
+        self.local_evaluator = CommonPolicyEvaluator(
+            self.env_creator,
+            self._default_policy_graph,
+            tf_session_creator=session_creator,
+            batch_mode="truncate_episodes",
+            observation_filter=self.config["observation_filter"],
+            env_config=self.config["env_config"],
+            model_config=self.config["model"],
+            policy_config=self.config
+            )
+        RemoteEvaluator = CommonPolicyEvaluator.as_remote(
             num_cpus=self.config["num_cpus_per_worker"],
-            num_gpus=self.config["num_gpus_per_worker"])(PPOEvaluator)
+            num_gpus=self.config["num_gpus_per_worker"])
         self.remote_evaluators = [
-            RemotePPOEvaluator.remote(
-                self.env_creator, self.config, self.logdir, True)
+            RemoteEvaluator.remote(
+                self.env_creator,
+                self._default_policy_graph,
+                batch_mode="truncate_episodes",
```

Comment:
complete_episodes is what it did originally right?

Reply:
thanks for the catch